### PR TITLE
Fix #509 Main menu Despawning text

### PR DESCRIPTION
**Pull request by TianQiBuTian**
And delete one superfluous "full stop"

### CODE REVIEW

**Comment by kvakvs** [comment]
Don't edit translation CSV files directly.
Your changes have to go to Crowdin.

**Comment by TianQiBuTian** [comment]
Done. But if doesn't change csv there will be bug. so need to changed translations file.

**Comment by kvakvs** [comment]
One of project admins can build and download CSV from Crowdin and add to your pull request.
I can do it later after work too.

File: TLM/TLM/UI/MainMenu/DespawnButton.cs [hunk 1]
```diff
@@ -11,8 +11,8 @@ public class DespawnButton : MenuButton {
 
         public override string Tooltip =>
             Options.disableDespawning
-                ? Translation.Options.Get("Maintenance.Button:Enable despawning")
-                : Translation.Options.Get("Maintenance.Checkbox:Disable despawning");
+                ? Translation.Menu.Get("Tooltip:Enable despawning")
+                : Translation.Menu.Get("Tooltip:Disable despawning");
```
Comment (kvakvs) [inline TLM/TLM/UI/MainMenu/DespawnButton.cs]: There is no string with key `Tooltip:Enable despawning` same for `Tooltip:Disable despawning`
It has to be introduced in Crowdin before we can accept this

**Comment by kvakvs** [comment]
@aubergine10 @krzychu124 please have a quick look, one of you should be enough to merge.
I have updated strings again, fixed the missing key, and added new string for keybind not set.

**Comment by aubergine10** [comment]
Will do some testing this evening.

**Comment by kvakvs** [comment]
Thanks for your contribution!

**Pull request merged**